CONTRIBUTING.md

```diff
@@ -288,26 +288,30 @@ node seed
 npm run develop
 ```
 
-### Setup freeCodeCamp via Docker
+### Setup freeCodeCamp using Docker
+#### Isolated
+Use this if you just want to work on freeCodeCamp.
 
-You need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing commands below.
+You'll need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing commands below.
```

Comment:
You'll need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing commands below.

**You will** need to have [docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed before executing **the** commands below.